dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/publish/servers/servlet40_incompatible/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2020, 2024 IBM Corporation and others.
```

Comment:
Why a 2020 copyright header for a new file?

Reply:
Was just copy pasted over. I'll delete it